README.md

```diff
@@ -1,2 +1,180 @@
-# ember-cli-workbox
-TBD
+ember-cli-workbox
+=================
+
+Service worker generator with precaching and some basic configurable options using [workbox-build](https://www.npmjs.com/package/workbox-build).
+
+For more details on Workbox check out:
+* [Workbox Google Developers](https://developers.google.com/web/tools/workbox/)
+* [Service Workers cookbook](https://serviceworke.rs/)
+
+Usage for Ember Cli
+-------------------
+
+`ember install ember-cli-workbox`
+
+### Configuration
+By default the service worker will be generated only for production builds and the service worker will be registered on an initializer.
+If you need to customize **ember-cli-workbox configuration** you can do it like this:
+
+```JavaScript
+//app/config/environment.js
+
+ENV['ember-cli-workbox'] = {
+  enabled: environment !== 'test',
```

Comment:
Use **2 whitespaces** to indent js blocks instead of **tabs**, the preview is ugly 😢